README.md

```diff
@@ -2,25 +2,36 @@
 
 ## Objectives
 
-1. Identify the default application layout.
-2. Yield to view templates from a layout.
-3. Specify a custom layout in ActionController on a controller level using the `layout` macro and on the action level using the `render :layout => "custom"` option.
+1. Identify the default application layout
+2. Yield to view templates from a layout
+3. Specify a custom layout in ActionController on a controller level using the `layout` macro and on the action level using the `render :layout => "custom"` option
 4. Use :layout => false to shut off the layout and only render the view
 
-## Outline
+## The Default Layout
```

Comment:
Before jumping into these super granular things, we want to give them the overarching "product vision". Make it fun, we want these to at least somewhat feel like they are actually building a thing